### PR TITLE
Adapt flags aliases to urfave/cli v2

### PR DESCRIPTION
**Pull request by feedmeapples**
## What was changed

Adapted flags aliases to urfave/cli v2 interface

## Why?

urfave/cli v2 flags interface has changed in v2. Fixed the aliases to work 

## Checklist

1. Closes 

2. How was this tested:

unit tested that there is no collision of aliases

3. Any docs updates needed?

### CODE REVIEW

**Review by yiminc — APPROVED** [review]

File: cli/batch.go [hunk 1]
```diff
@@ -39,7 +39,8 @@ func newBatchCommands() []*cli.Command {
 			Usage: "Describe a batch operation job",
 			Flags: []cli.Flag{
 				&cli.StringFlag{
-					Name:     FlagJobIDWithAlias,
+					Name:     FlagJobID,
+					Aliases:  FlagRunIDAlias,
```
Comment (feedmeapples) [inline cli/batch.go:43]:
```suggestion
					Aliases:  FlagJobIDAlias,
```

File: cli/flags.go [hunk 2]
```diff
@@ -31,203 +31,195 @@ import (
...
+	FlagMinEventID                    = "min-event-id"
+	FlagMaxEventID                    = "max-event-id"
+	FlagStartEventVersion             = "start-event-version"
+	FlagTaskQueue                     = "task-queue"
+	FlagTaskQueueAlias                = []string{"tq"}
+	FlagTaskQueueType                 = "task-queue-type"
+	FlagTaskQueueTypeAlias            = []string{"tqt"}
+	FlagWorkflowIDReusePolicy         = "workflow-id-reuse-policy"
+	FlagCronSchedule                  = "cron"
+	FlagWorkflowType                  = "type"
```
Comment (feedmeapples) [inline cli/flags.go:64]:
```suggestion
	FlagWorkflowType                  = "workflow-type"
```

File: cli/flags.go [hunk 3]
```diff
@@ -31,203 +31,195 @@ import (
...
+	FlagWorkflowTypeAlias             = []string{"t"}
```
[the remark this answers is not in the document]
Comment (feedmeapples) [reply cli/flags.go:65]:
```suggestion
	FlagWorkflowType                  = "type"
	FlagWorkflowTypeAlias             = []string{"wt"}
```

File: cli/flags.go [hunk 4]
```diff
@@ -31,203 +31,195 @@ import (
...
+	FlagMaxEventID                    = "max-event-id"
+	FlagStartEventVersion             = "start-event-version"
+	FlagTaskQueue                     = "task-queue"
+	FlagTaskQueueAlias                = []string{"tq"}
+	FlagTaskQueueType                 = "task-queue-type"
+	FlagTaskQueueTypeAlias            = []string{"tqt"}
+	FlagWorkflowIDReusePolicy         = "workflow-id-reuse-policy"
+	FlagCronSchedule                  = "cron"
+	FlagWorkflowType                  = "workflow-type"
+	FlagWorkflowTypeAlias             = []string{"t"}
```
Comment (feedmeapples) [inline cli/flags.go:65]:
```suggestion
	FlagWorkflowTypeAlias             = []string{"wt"}
```

File: cli/flags.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (feedmeapples) [reply cli/flags.go:65]: imo this makes typing the flags way too cumbersome, especially given that the auto-completion is often not enabled (or not even available). There are also other flags with a similar case, raises a question when do we include/exclude having "workflow" in the flag:

- "status" (workflow status)
- "execution-timeout"
- "run-timeout"


In CLIs having short flags is pretty fine. Especially for common flags and given that the auto-completion is not always enabled. Then having a proper --help output will still let them figure out the usage

**Pull request merged**